plugins/kubernetes/src/components/StatefulSetsAccordions/StatefulSetDrawer.test.tsx

```diff
@@ -28,19 +28,25 @@ describe('StatefulSetDrawer', () => {
       />,
     );
 
-    expect(getAllByText('dice-roller')).toHaveLength(3);
+    expect(getAllByText('dice-roller')).toHaveLength(4);
     expect(getByText('StatefulSet')).toBeInTheDocument();
     expect(getByText('YAML')).toBeInTheDocument();
-    expect(getByText('Type: RollingUpdate')).toBeInTheDocument();
+    expect(
+      getByText(textContentMatcher('Type: RollingUpdate')),
+    ).toBeInTheDocument();
     expect(getByText('Rolling Update:')).toBeInTheDocument();
-    expect(getByText('Max Surge: 25%')).toBeInTheDocument();
-    expect(getByText('Max Unavailable: 25%')).toBeInTheDocument();
+    expect(getByText(textContentMatcher('Max Surge: 25%'))).toBeInTheDocument();
+    expect(
+      getByText(textContentMatcher('Max Unavailable: 25%')),
+    ).toBeInTheDocument();
     expect(getByText('Pod Management Policy')).toBeInTheDocument();
     expect(getByText('Parallel')).toBeInTheDocument();
     expect(getByText('Service Name')).toBeInTheDocument();
     expect(getByText('Selector')).toBeInTheDocument();
     expect(getByText('Match Labels:')).toBeInTheDocument();
-    expect(getByText('App: dice-roller')).toBeInTheDocument();
+    expect(
+      getByText(textContentMatcher('App: dice-roller')),
```

Comment:
Btw why was it necessary to add these matchers?

Reply:
Because of adding the Typography elements to [packages/core-components/src/components/StructuredMetadataTable/StructuredMetadataTable.tsx](https://github.com/backstage/backstage/pull/14671/files#diff-afe5c5c4fe73e4feda4d0398acd5e73dce34549c80db52c21b3df6460f62b799R92-R132) the text of those tests are now split into 2 different HTML elements, to keep the test close to the same I introduced the matcher, otherwise would have to try and match each part of the text separately.